docs/src/operations/requirements.md

```diff
@@ -97,7 +109,7 @@ Prebuilt binaries are available for Linux x86_64 on CPUs supporting AVX2 \(Ubunt
 MacOS or WSL users may build from source.
 
 ## Networking
-Internet service should be at least 1GBbit/s symmetric, commercial. 10GBit/s preferred.
+Internet service should be at least 1GBbit/s symmetric, commercial. 10GBit/s preferred (especially for mainnet).
```

Comment:
```suggestion
Internet service should be at least 1GBbit/s symmetric, commercial. 10GBit/s preferred (especially for mainnet-beta).
```

Reply:
Fixed in cf8d6dbe7891746c9b46d8f173d9424687961d72